fast64_internal/oot/oot_level_classes.py

```diff
@@ -443,6 +444,23 @@ def hasAlternateHeaders(self):
             and len(self.cutsceneHeaders) == 0
         )
 
+    def getObjectLengthDefineName(self, headerIndex: int):
+        return f"LENGTH_{self.objectListName(headerIndex).upper()}"
+
+    def getActorLengthDefineName(self, headerIndex: int):
+        return f"LENGTH_{self.actorListName(headerIndex).upper()}"
+
+    def getObjectLengthDefine(self, headerIndex: int):
+        return f"#define {self.getObjectLengthDefineName(headerIndex)} {len(self.objectIDList)}\n"
```

Comment:
I think OOTRoom shouldn't be responsible for writing C, but also `getObjectLengthDefine` is only used once, so I'd put this code directly where it's used instead of `getObjectLengthDefine`